defaults/main.yml

```diff
@@ -2,18 +2,18 @@
 # defaults file for ansible-role-sjk/
 
 # SJK binary version.
-# LATEST or specific version number.
-# Default value: LATEST
+# latest or specific version number.
+# Default value: latest
 # See https://mvnrepository.com/artifact/org.gridkit.jvmtool/sjk for available versions
-sjk_version: LATEST
+sjk_version: latest
 
 # true: use the full sjk binary (with mxdump)
 # false: use lighter sjk binary (without mxdump)
 # Default value: false
 sjk_use_sjkplus: false
 
 # SJK Binary destination path
-sjk_dest: '/opt/sjk/sjk-{{ sjk_version | lower }}'
+sjk_dest: '/opt/sjk/sjk{% if sjk_use_sjkplus is defined and sjk_use_sjkplus== true %}-plus{% endif %}-{{ sjk_version | lower }}'
```

Comment:
good practice is to use space before and after operators: `sjk_use_sjkplus== true` -> `sjk_use_sjkplus == true`

Reply:
sure. done